source/deploy/high-scale-troubleshoot.rst

```diff
@@ -12,4 +12,18 @@ My system keeps hanging when I search for a message in Mattermost
 
 First, check how many messages have been posted on your system, including deleted posts and posts made using automations.
 
-Go to the **System Console > Reporting > Site Statistics** and review the **Total Posts** figure reported. If this figure is above 3,000,000 posts, we recommend deploying Elasticsearch alongside your Mattermost server for improved search performance. Follow our guides to `deploy an Elasticsearch server </scale/elasticsearch.html>`__.
\ No newline at end of file
+Go to the **System Console > Reporting > Site Statistics** and review the **Total Posts** figure reported. If this figure is above 3,000,000 posts, we recommend deploying Elasticsearch alongside your Mattermost server for improved search performance. Follow our guides to `deploy an Elasticsearch server </scale/elasticsearch.html>`__.
+
+How to troubleshoot server-side performance issues without Prometheus or Grafana?
+-----------------------------------------------------------------------------------
+
+Enable slow query logging in PostgreSQL and leave it enabled to gather data over time. PostgreSQL's slow query log helps you identify queries that take longer than a specified amount of time. The slow query log isn't enabled by default, and must be enabled manually.
+
+To enable slow query logging globally, change the following line in postgresql.conf and reload the page: ``log_min_duration_statement = 5000``. When you this value to ``5000``, PostgreSQL considers queries that take longer than 5 seconds to be slow queries and logs them in the log file.
+
+To enable slow query logging for a specific database, use ``ALTER DATABASE`` to change the configuration parameter for a single database. For example:
+
+.. code-block:: none
+
+  postgres=# ALTER DATABASE test SET log_min_duration_statement = 5000;
```

Comment:
let's change `test` to `mattermost` and make a call out that the database name should be what you call it. 99% will be `mattermost`